### ISSUE TITLE
How to Fetch Whether an Outlook Mail is "Confidential", "Encrypt Only", or "Do Not Forward" Using Office API or Other Methods?

### ISSUE THREAD

**Issue by AacharanJain** [OfficeDev/office-js#5003]
Hello,

We are currently building an Outlook add-in using React, and we need to determine if the currently opened email is marked as "Confidential" or "Do Not Forward".We want to check for the following encryption types:

- Encrypt Only
- Confidential
- Highly Confidential
- Do Not Forward

Our use case requires certain functionalities in our add-in to be restricted based on whether the email is marked as "Confidential" or "Do Not Forward."

We have attempted to use both the Office.js API and Graph API to retrieve any attributes that would help us identify the email's encryption status. However, we haven't been able to find a way to access this information.

Could anyone provide guidance or suggestions on how to fetch the encryption status ("Encrypt Only", "Confidential", "Highly Confidential", "Do Not Forward") of an email using the Office.js API or Graph API? Any insights would be greatly appreciated!

Thank you in advance for your help!

**Comment by patilganesh-msft** [comment]
Office.context.mailbox.item
=> sensitivityLabel

**Comment by AacharanJain** [comment]
Hi @patilganesh-msft ,

The sensitivityLabel is associated with the sensitivity level of an email, such as Normal, Personal, Private, or Confidential. I've attached a screenshot for reference.
![image](https://github.com/user-attachments/assets/e10b125a-db1f-47a9-8c8d-6af810bfe2d8)


But, we're looking for an attribute that specifies the encryption type of the email, such as Encrypt Only, Do Not Forward, Confidential, and Highly Confidential. A screenshot has also been attached for reference.
![Capture11](https://github.com/user-attachments/assets/4992652b-f27e-45e9-86c6-75b0bc60fbe9)


We've tried the approach you suggested, but it doesn’t meet our requirements.

**Office.context.mailbox.item.sensitivity**: This consistently **returns undefined** for all email types.

![image](https://github.com/user-attachments/assets/bea355bf-b8ed-4797-9261-19a9f8847445)


We are still looking for a solution that meets our requirements. Thanks for the response and any additional suggestions would be greatly appreciated.
